include/triton/Conversion/TritonGPUToLLVM/PatternTritonGPUOpToLLVM.h

```diff
@@ -120,7 +120,12 @@ void populatePrintOpToLLVMPattern(LLVMTypeConverter &typeConverter,
                                   RewritePatternSet &patterns,
                                   const TargetInfoBase &targetInfo,
                                   PatternBenefit benefit);
-
+namespace proton {
```

Comment:
It would be better to move it to the Proton directory. Then each backend invokes the function independently in their `TritonGPUToLLVM.cpp`

Reply:
In this way each backend can also have their own lower logic if inline assembly is used

Reply:
Updated